client/components/common/AtAndBrowserDetailsModal/index.jsx

```diff
@@ -90,15 +95,13 @@ const AtAndBrowserDetailsModal = ({
         }
 
         const foundBrowserVersion =
-            uaBrowser === browserName &&
-            uaMajor !== '0' &&
-            `${uaMajor}.${uaMinor}.${uaPatch}`;
+            uaBrowser === browserName && uaMajor !== '0' && `${uaMajor}`;
 
         if (
             // don't force browserVersion update with admin (unless first run)
             (!isAdmin || (isAdmin && firstLoad)) &&
             // check that saved browserVersion is the same as detected
-            !browserVersion.includes(`${uaMajor}.${uaMinor}.${uaPatch}`) &&
+            !browserVersion.includes(`${uaMajor}`) &&
```

Comment:
Leaving as the string template does make it more intentional but the hook does return `ua...` values as string if you'd like to simplify this.

The function would also implicitly convert a numeric value to string as well. But this is a nit!

```suggestion
            !browserVersion.includes(uaMajor) &&
```